### PR TITLE
Add `strings` example

### PR DESCRIPTION
**Pull request by maxwellE**
Part of #72, #78, and #96. 

Adds a usage of `strings` with an example

### CODE REVIEW

File: xcodeproj/internal/top_level_targets.bzl [hunk 1]
```diff
@@ -235,6 +235,15 @@ def process_top_level_target(
...
 
+    if automatic_target_info.strings:
+        additional_files.extend(
+            getattr(
+                ctx.rule.files,
+                automatic_target_info.strings,
+                [],
+            ),
+        )
+
```
Comment (brentleyjones) [inline xcodeproj/internal/top_level_targets.bzl:246]: I believe strings are handled by the resource management. I think all of the handling can be removed, and we only need the example. You can confirm in the BwX example at compile and runtime?

Comment (maxwellE) [reply xcodeproj/internal/top_level_targets.bzl:246]: Screen Shot 2022-09-27 at 4 59 19 PM

Seeing this after I remove all the handling logic and just leave in the edits to the example

Comment (brentleyjones) [reply xcodeproj/internal/top_level_targets.bzl:246]: Yep, that is just files in the `extra_files`, you should be able to see them still in the `resources` sections.

Comment (maxwellE) [reply xcodeproj/internal/top_level_targets.bzl:246]: Screen Shot 2022-09-27 at 5 18 05 PM

You are right! Missed those at first glance

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**